### DISCUSSION TITLE
24.03.11 Development Meeting

### DISCUSSION THREAD

**Discussion by Woolfrey** [Woolfrey/software_robot_library#99] [Meetings]
#### Chair:
- Woolfrey @woolfrey

#### Attendees:
- Sheila @ssutjipto

___________________

## Announcements
_Any relevant news._

## Agenda
_Note: Last week's actions become this week's agenda_

| Item | Responsible | Issue | Outcome |
|-------|------------------|----------|--------------|

N/A

## Additional Discussion
_Anything not captured in the agenda._

| Topic | Raised By | Issue | Outcome |
|--------|----------------|---------|--------------|
| Need to develop `Action` server | @Woolfrey |  [Link](https://github.com/Woolfrey/interface_serial_link/issues/1) | Need to list all the fundamental `Action`sfor a serial link robot. Need an informative name, goals, result, and feedback. |
| Need to merge `template` branch in to `main`/`master` | @Woolfrey | #91 | @ssutjipto  to check dynamics output |

## Actions

| Action | Responsible | Tag |
|----------|------------------|-------|
| List and agree on`Action` definitions for a serial link robot | @Woolfrey @ssutjipto | [Link](https://github.com/Woolfrey/interface_serial_link/issues/1) |
| Check output of `template` branch and merge | @ssutjipto | #91 |
| Plot outputs of `Trajectory` classes and ensure correct | @Woolfrey | #100  |
| Link `interface_serial_link` repository to project | @Woolfrey | N/A |

**Comment by ssutjipto** [comment]
For the action definitions, the existing actions in the [control_msgs repository](https://github.com/ros-controls/control_msgs/tree/master/control_msgs/action) may be a good start? 
Was there anything that you had in mind which hasn't been covered by these actions?

**Reply by Woolfrey** [reply]
@ssutjipto  WHY ARE YOU AWAKE???
